STYLEGUIDE.md

```diff
@@ -0,0 +1,25 @@
+## Conda Style Guide
+
+In an effort to decrease the context switching/learning barrier between the different projects within the Conda org, a standard set of code styles is defined. The intention of these guidelines is not to overburden or complicate the contribution process; if any style starts to become a burden, its inherent benefit should be reevaluated.
+
+> ⚠️ See conda-incubator/ceps#24 for our policy regarding the Python versions Conda projects need to support. Conda plugins must also support this range of versions in order to continue to be of value to current users.
```

Comment:
This may actually be too early to leave this here, even though I understand why you'd want it. Until the CEP has been voted on, it's really only a draft policy and we should be careful about calling it our policy to allow full transparency about the CEP process. Would you mind updating the text to say it's a draft currently, or just remove it until we've finalized and voted on it?

Reply:
Hehe, I had hoped that those CEPs would've gotten some more traction by this time, I'll remove it for now.